march_hardware/src/March4.cpp

```diff
@@ -15,18 +15,40 @@ namespace march4cpp
 {
 MARCH4::MARCH4()
 {
-  TemperatureGES temperatureGES = TemperatureGES(1, 0);
-  // TODO(Isha, Martijn) double-check these numbers.
-  Encoder enc = Encoder(16, 37961, 59649, 39717, 0.3);
-  IMotionCube imc = IMotionCube(2, enc);
-  // For now we are only running on the test joint
-  // TODO(Isha) refactor this that you can switch between test_joint, MARCH4 (and MARCH3)
-  Joint temp = Joint("test_joint", temperatureGES, imc);
+  Encoder RHJenc = Encoder(16, 22134, 43436, 24515, 0.05);
+  Encoder LHJenc = Encoder(16, 9746, 31557, 11830, 0.05);
+
```

Comment:
What does each number mean and why do they need to be this exact number? Same question for the IMC numbers

Reply:
Check the constructor, will become obsolete by march_hardware_builder.